.cicd/scripts/srw_ftest.sh

```diff
@@ -115,16 +130,22 @@ export OMP_NUM_THREADS=1
 )
 set +x
 
-echo "# Try the first few simple SRW tasks ..."
 results_file=${workspace}/functional_test_results_${SRW_PLATFORM}_${SRW_COMPILER}.txt
 rm -f ${results_file}
+
 status=0
-for task in ${TASKS[@]:0:${TASK_DEPTH}} ; do
+
+# Limit to machines that are fully ready
+target_machines=( jet cheyenne noaacloud )
+if [[ ${target_machines[@]} =~ ${platform,,} ]] ; then
+    echo "# Try the first few simple SRW tasks ..."
+    for task in ${TASKS[@]:0:${TASK_DEPTH}} ; do
                 echo -n "./$task.sh ... "
                 ./$task.sh > $task-log.txt 2>&1 && echo "COMPLETE" || echo "FAIL rc=$(( status+=$? ))"
                 # stop at the first sign of trouble ...
                 [[ 0 != ${status} ]] && echo "$task: FAIL" >> ${results_file} && break || echo "$task: COMPLETE" >> ${results_file}
-done
+    done
+fi
```

Comment:
The `fi` on line 148 needs to drop to the last line of the script.  If this doesn't happen, then the script will fail with:

`grep: /scratch2/NAGAPE/epic/Michael.Lueken/ufs-srweather-app/functional_test_results_hera_intel.txt: No such file or directory`

Reply:
let's make sure there is a results_file

Reply:
Do we want or need a results_file for Hera?  The tests aren't being run on the machine since the system won't let us change the ulimit.

Reply:
The idea behind the 
```
target_machines=( jet cheyenne noaacloud )
if [[ ${target_machines[@]} =~ ${platform,,} ]] ; then
```
is to only run the script on the machines that they will work on.  This beginning logic should be placed around line 51.

Reply:
I've gone ahead and resubmitted the Jenkins tests.  Having said that, the logic present on line 141 ensures that the test won't run on Hera.  I don't see the point in creating a blank file.  Once the ability to manipulate the ulimit has been made on Hera, then we can remove the logic, but until that happens, there is no reason for any part of this script to be executed on that machine.

Reply:
A more robust solution is that we use a deny_machines list, so that even non-tier1 can run this script manually.